src/pages/_app.js

```diff
@@ -0,0 +1,17 @@
+import '../style.css';
+import Layout from '../components/Layout';
+import { ThemeProvider, createMuiTheme } from '@material-ui/core/styles';
+
+import appTheme from "../theme";
+
+function TreetrackerApp({ Component, pageProps }) {
+  return (
+    <ThemeProvider theme={appTheme}>
+      <Layout>
+        <Component {...pageProps} />
+      </Layout>
+    </ThemeProvider>
+  );
+}
+
+export default TreetrackerApp;
```

Comment:
@cpaule1811 I don't get it, what's this file for? we have had the App.js in the root folder, and your `page/index.js` is importing App.js rather than this _app.js right?

Reply:
The app component initialises all the page components. This document is used to alter the behaviour of that initialisation. Things like providers and state that need to be used accross the application can be placed in this document. the <component {...pageProps} /> is passed from each page file depending on the route. 

As part of Next syntax the document must be named _app.js. 

Hope that clears it up.